### PR TITLE
🌐[i18n-KO] Translated `llm_tutorial.md` to Korean

### PR DESCRIPTION
**Pull request by harheem**
# What does this PR do?

Translated the `llm_tutorial.md` file of the documentation to Korean 😄 
Thank you in advance for your review!

Part of https://github.com/huggingface/transformers/issues/20179

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [x] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)
@bolizabeth, @nuatmochoi, @heuristicwave, @mjk0618, @keonju2, @harheem, @HongB1, @junejae, @54data, @seank021, @augustinLib, @sronger, @TaeYupNoh, @kj021, @eenzeenee

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review? (Final)
 May you please review this PR?
@sgugger, @ArthurZucker, @eunseojo

### CODE REVIEW

File: docs/source/ko/llm_tutorial.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+>>> tokenizer.batch_decode(generated_ids, skip_special_tokens=True)[0]
+'I am a cat. I am a cat. I am a cat. I am a cat'
+
+>>> # With sampling, the output becomes more creative!
+>>> generated_ids = model.generate(**model_inputs, do_sample=True)
+>>> tokenizer.batch_decode(generated_ids, skip_special_tokens=True)[0]
+'I am a cat.\nI just need to be. I am always.\nEvery time'
+```
+
+### 잘못된 패딩[[wrong-padding-side]]
```
Comment (harheem) [inline docs/source/ko/llm_tutorial.md:172]:
```suggestion
### 잘못된 패딩 [[wrong-padding-side]]
```

File: docs/source/ko/llm_tutorial.md [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+LLM과 자기회귀 생성을 함께 사용할 때 핵심적인 부분은 이 확률 분포로부터 다음 토큰을 어떻게 고를 것인지입니다. 다음 반복 과정에 사용될 토큰을 결정하는 한, 어떠한 방법도 가능합니다. 확률 분포에서 가장 가능성이 높은 토큰을 선택하는 것처럼 간단할 수도 있고, 결과 분포에서 샘플링하기 전에 수십 가지 변환을 적용하는 것처럼 복잡할 수도 있습니다.
+
+<!-- [GIF 2 -- TEXT GENERATION] -->
+<figure class="image table text-center m-0 w-full">
+    <video
+        style="max-width: 90%; margin: auto;"
+        autoplay loop muted playsinline
+        src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/blog/assisted-generation/gif_2_1080p.mov"
+    ></video>
+    <figcaption>"자동 회귀 생성은 확률 분포에서 다음 토큰을 반복적으로 선택하여 텍스트를 생성합니다."</figcaption>
```
Comment (harheem) [inline docs/source/ko/llm_tutorial.md:62]:
```suggestion
    <figcaption>"자기회귀 생성은 확률 분포에서 다음 토큰을 반복적으로 선택하여 텍스트를 생성합니다."</figcaption>
```

**Review by nuatmochoi — CHANGES_REQUESTED** [review]

File: docs/source/ko/llm_tutorial.md [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+시작하기 전에 필요한 모든 라이브러리가 설치되어 있는지 확인하세요:
+
+```bash
+pip install transformers bitsandbytes>=0.39.0 -q
+```
+
+
+## 텍스트 생성 [[generate-text]]
+
+[인과적 언어 모델링(causal language modeling)](tasks/language_modeling)을 목적으로 학습된 언어 모델은 일련의 텍스트 토큰을 입력으로 사용하고, 그 결과로 다음 토큰이 나올 확률 분포를 제공합니다.
```
Comment (harheem) [inline docs/source/ko/llm_tutorial.md:41]: NLP 스터디 책에서는 "코잘 언어 모델링" 이라고 표기하였기에 어떤것을 사용할지 고민입니다 👀

File: docs/source/ko/llm_tutorial.md [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+
+>>> # Setting `max_new_tokens` allows you to control the maximum length
+>>> generated_ids = model.generate(**model_inputs, max_new_tokens=50)
+>>> tokenizer.batch_decode(generated_ids, skip_special_tokens=True)[0]
+'A sequence of numbers: 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16,'
+```
+
+### 잘못된 생성 모드 [[incorrect-generation-mode]]
+
+기본적으로 [`~generation.GenerationConfig`] 파일에서 별도로 지정하지 않으면, `generate`는 각 반복에서 가장 확률이 높은 토큰을 선택합니다(그리디 디코딩). 하려는 작업에 따라 이 방법은 바람직하지 않을 수 있습니다. 예를 들어, 챗봇이나 에세이 작성과 같은 창의적인 작업은 샘플링이 적합할 수 있습니다. 반면, 오디오 트랜스크립션이나 번역과 같은 입력 기반 작업은 그리디 디코딩이 더 적합할 수 있습니다. `do_sample=True`로 샘플링을 활성화할 수 있으며, 이 주제에 대한 자세한 내용은 이 [블로그 포스트](https://huggingface.co/blog/how-to-generate)에서 볼 수 있습니다.
```
[the remark this answers is not in the document]
Comment (harheem) [reply docs/source/ko/llm_tutorial.md:152]: `오디오를 텍스트로 변환하거나` 좋은 표현 인 것 같아요! 감사합니다.

File: docs/source/ko/llm_tutorial.md (hunk 3 above)
Comment (nuatmochoi) [reply docs/source/ko/llm_tutorial.md:41]: 제가 번역한 타 문서([pert](https://github.com/nuatmochoi/transformers/blob/94673840973f8b5fd41bc88daf1e4ad75708afa8/docs/source/ko/peft.md#peft-%EC%96%B4%EB%8C%91%ED%84%B0-%EA%B0%80%EC%A0%B8%EC%98%A4%EA%B8%B0-load-a-peft-adapter))에서는`인과 관계 언어 모델링`으로 번역했었습니다. 저는 `인과적 언어 모델링`, `인과 언어 모델링` 모두 괜찮을 것 같아 glossary에 후보군으로 추가해놓았는데요, 어떤 게 더 좋을지 토의해보면 좋을 것 같습니다:)

File: docs/source/ko/llm_tutorial.md [hunk 5]
```diff
@@ -0,0 +1,221 @@
...
+
+>>> # With sampling, the output becomes more creative!
+>>> generated_ids = model.generate(**model_inputs, do_sample=True)
+>>> tokenizer.batch_decode(generated_ids, skip_special_tokens=True)[0]
+'I am a cat.\nI just need to be. I am always.\nEvery time'
+```
+
+### 잘못된 패딩[[wrong-padding-side]]
+
+LLM은 [디코더 전용](https://huggingface.co/learn/nlp-course/chapter1/6?fw=pt) 구조를 가지고 있어, 입력 프롬프트에 대해 지속적으로 반복 처리를 합니다. 입력 데이터의 길이가 다르면 패딩 작업이 필요합니다. LLM은 패딩 토큰에서 작동을 이어가도록 설계되지 않았기 때문에, 입력 왼쪽에 패딩이 추가 되어야 합니다. 그리고 어텐션 마스크도 꼭 `generate` 함수에 전달되어야 합니다.
```
Comment (0525hhgus) [inline docs/source/ko/llm_tutorial.md:174]:
```suggestion
LLM은 [디코더 전용](https://huggingface.co/learn/nlp-course/chapter1/6?fw=pt) 구조를 가지고 있어, 입력 프롬프트에 대해 지속적으로 반복 처리를 합니다. 입력 데이터의 길이가 다르면 패딩 작업이 필요합니다. LLM은 패딩 토큰에서 작동을 이어가도록 설계되지 않았기 때문에, 입력 왼쪽에 패딩이 추가 되어야 합니다. 그리고 어텐션 마스크도 꼭 `generate` 함수에 전달되어야 합니다!
```
원문에서 느낌표(!)가 있어서, 추가했습니다!

**Review by 0525hhgus — CHANGES_REQUESTED** [review]
새로 추가된 문서지만 꼼꼼하게 번역해주셨군요! 좋은 번역 감사합니다 👍 
아래 짧은 리뷰 남겼으며, 다른 리뷰 사항은 없습니다 😄

File: docs/source/ko/llm_tutorial.md [hunk 6]
```diff
@@ -0,0 +1,221 @@
...
+이어서 텍스트 입력을 [토크나이저](tokenizer_summary)으로 전처리하세요.
+
+```py
+>>> from transformers import AutoTokenizer
+
+>>> tokenizer = AutoTokenizer.from_pretrained("openlm-research/open_llama_7b")
+>>> model_inputs = tokenizer(["A list of colors: red, blue"], return_tensors="pt").to("cuda")
+```
+
+`model_inputs` 변수에는 토큰화된 텍스트 입력과 함께 어텐션 마스크가 들어 있습니다. [`~generation.GenerationMixin.generate]`는 어텐션 마스크가 제공되지 않았을 경우에도 이를 추론하려고 노력하지만, 최상의 성능을 위해서는 가능하면 어텐션 마스크를 전달하는 것을 권장합니다.
```
Comment (stevhliu) [inline docs/source/ko/llm_tutorial.md:104]:
```suggestion
`model_inputs` 변수에는 토큰화된 텍스트 입력과 함께 어텐션 마스크가 들어 있습니다. [`~generation.GenerationMixin.generate`] 는 어텐션 마스크가 제공되지 않았을 경우에도 이를 추론하려고 노력하지만, 최상의 성능을 위해서는 가능하면 어텐션 마스크를 전달하는 것을 권장합니다. 
```

**Review by stevhliu — APPROVED** [review]
Great work! Just one small typo to fix and then we're ready to merge! 😄

File: docs/source/ko/llm_tutorial.md (hunk 6 above)
Comment (harheem) [reply docs/source/ko/llm_tutorial.md:104]: Ah, I must have missed that. Thank you so much for the review! 🥹

**Pull request merged**